assets/js/atcb.js

```diff
@@ -566,6 +570,9 @@ function atcb_generate_yahoo(data) {
   }
   if (data['description'] != null && data['description'] != '') {
     url += '&desc=' + encodeURIComponent(data['description']);
+  } 
+  if (data['recurrence'] != null && data['recurrence'] != '') {
```

Comment:
Codacy has a fix for the issue: [Replace `'recurrence']·!=·null·&&·data['recurrence']·!=·''` with `"recurrence"]·!=·null·&&·data["recurrence"]·!=·""`](https://app.codacy.com/gh/jekuer/add-to-calendar-button/pullRequest?prid=9703945)
```suggestion
  if (data["recurrence"] != null && data["recurrence"] != "") {
```